build.yaml

```diff
@@ -1021,6 +1029,7 @@ libs:
   - grpc++_codegen_base_src
   - grpc++_codegen_proto
   - grpc++_config_proto
+  - thrift_util
```

Comment:
Strongly oppose anything that would require thrift installation as part of the C/C++ build-install-test requirements.